### ISSUE TITLE
Advanced Search - Accessibility Updates

### ISSUE THREAD

**Issue by sshetenhelm** [yalelibrary/YUL-DC#3195]
**Story**
From #3068 

Page does not start with a level 1 heading: "Advanced Search" is a level 1 heading, but the heading that is being read first is the h2 in the nav bar. Headings should be used for page structure, and not for styling. Use ul or li tags for navbar list items and style with CSS. This will allow screen readers to see the actual level 1 heading ("Advanced Search").

As per Tracy,  “Ideally, the Yale wordmark should not be an image”; however, we were specifically requested to use this image. We will add alt text.

Broken ARIA reference: all/any sort field, all search form fields (An element has an aria-labelledby or aria-describedby value that does not match the id attribute value of another element in the page)

**Acceptance**

- [x] Use ul or li tags for navbar list items and style with CSS. This will allow screen readers to see the actual level 1 heading ("Advanced Search").

- [x] Add alt text to Yale Library logo link (navbar-logo): alt=“Yale Library Home”

- [x] Update form elements to use labels - As per Tracy: 

Do not use "arialabelled-by" for form elements; use labels instead: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby (The drop-down select at the top of the page should also have a label, not an aria-describedby attribute - this attribute can be removed as it does not point to anything)

If using "aria-labelledby" in the future, fix the typo. In our code it is "aria-labeledby", which is not a valid element

**Comment by K8Sewell** [comment]
Looking good on Test.  Will promote to UAT.

<details>
* Uses <ul> element instead of <h2>
<img width="909" height="1094" alt="Image" src="https://github.com/user-attachments/assets/6ff7418e-4515-462a-88be-7d054e7c2720" />

* Adds alt tag to logo

<img width="958" height="1096" alt="Image" src="https://github.com/user-attachments/assets/3043c7e0-657a-4232-8715-5095df27ca4c" />


* Adds label tags to form

<img width="1545" height="986" alt="Image" src="https://github.com/user-attachments/assets/837c6d9c-3ce0-4a7a-975e-a3a300aba3b4" />

</details>
